federation/cluster/federation-up.sh

```diff
@@ -82,7 +82,8 @@ function init() {
       "${FEDERATION_NAME}" \
       --host-cluster-context="${HOST_CLUSTER_CONTEXT}" \
       --dns-zone-name="${DNS_ZONE_NAME}" \
-      --image="${kube_registry}/hyperkube-amd64:${kube_version}"
+      --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
+      --apiserver-arg-overrides="--storage-backend=etcd2"
```

Comment:
Why would it break a user who is running `kubefed init`? `kubefed init` brings up a new control plane and people who bring up new control planes can choose `etcd3`. We don't have upgrade support yet. 

Should we also test with `etcd3`? Yes. And that's coming. I am asking people to not make too many disruptive changes at this point. We are trying really hard to get the CI tests to a green state.